ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Make sure you try the following before submitting the issue, thank you!
+
+- Check that umake is the latest version (16.11.1)
+- If you're not running the latest development version of Ubuntu, zesty (17.04), or a distro based on zesty, add the ppa as described in the README.
```

Comment:
Same, don't specify any version (and there is no distro based on development version normally).